editions/tw5.com/tiddlers/gettingstarted/GettingStarted.tid

```diff
@@ -15,7 +15,7 @@ Brief instructions for getting started on the different platforms and configurat
 
 Troubleshooting:
 
-* Don't attempt to use the browser ''File''/''Save'' menu option to save changes (it doesn't work)
+* Don't attempt to use the browser ''File''/''Save'' menu option to save changes (it doesn't work).
```

Comment:
The full stop is probably missing on purpose. see: https://github.com/Jermolene/TiddlyWiki5/issues/519